web/vtadmin/src/components/VitessLogo.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
These changes are also the same in OS VTAdmin: https://github.com/vitessio/vitess/pull/9866